.github/workflows/ci.yaml

```diff
@@ -0,0 +1,155 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - 'release-*'
+  pull_request:
+    branches:
+      - master
+      - 'release-*'
+env:
+  BUILDAH_VERSION: v1.14.0
+  RUNC_VERSION: v1.0.0-rc10
+  CRIO_VERSION: v1.17.0
+  GO_VERSION: 1.13.8
+jobs:
+
+  tools:
+    name: Build Buildah and RunC
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/setup-go@v2-beta
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: Set up environment
+        shell: bash
+        run: |
+          echo "::set-env name=GO_PATH::$(go env GOPATH)"
+      - uses: actions/cache@v1
+        id: cache-buildah
+        with:
+          path: ${{ env.GO_PATH }}
+          key: ${{ runner.os }}-buildah-${{ env.BUILDAH_VERSION }}
+      - name: Build tools
+        if: steps.cache-buildah.outputs.cache-hit != 'true'
+        run: |
+          sudo apt-get update
+          sudo apt-get -y install e2fslibs-dev libfuse-dev libgpgme11-dev libdevmapper-dev libglib2.0-dev libprotobuf-dev
+          # build buildah
+          mkdir -p ${GO_PATH}/src/github.com/containers
+          cd ${GO_PATH}/src/github.com/containers
+          git clone --single-branch --depth 1 -b ${BUILDAH_VERSION} https://github.com/containers/buildah
+          cd buildah
+          make buildah TAGS=""
```

Comment:
We can install buildah instead of building. It's explained here: https://github.com/containers/buildah/blob/master/install.md#ubuntu

Reply:
Thanks! This way we can drop the buildah job completely. Now all jobs run in parallel.